src/Markdown/Parser.php

```diff
@@ -41,6 +42,10 @@ public function converter(): CommonMarkConverter
             $env->addExtension($ext);
         }
 
+        foreach ($this->renderers() as $ren) {
+            $env->addRenderer(...$ren);
+        }
+
```

Comment:
no need to shorten this, can we use `renderer`?

Reply:
Fair, I wanted to stay consistent with the code extension but `ren` definitely isn't self explanatory